apps/are_form/views/appealstages.html

```diff
@@ -38,7 +38,7 @@ <h2 class="appeal-stage-header govuk-details__summary-text">
 </div>
 
 <div id="reference-list" class="header-context">
-  <ol role="guidance" class="group">
+  <ol role="navigation" class="group">
     <li><strong>Guidance</strong></li>
     <li><a href="{{#url}}/guidance{{/url}}" class="govuk-link">{{#t}}pages.reference-links.guidance{{/t}}</a></li>
```

Comment:
Hi @tobyschaefferHO just following in from Elias's comment, we can also add the role of link to the links on the page.

Reply:
Hi @tobyschaefferHO yes I think we can add the role="link" seeing as its being used in other HoF forms.